### PR TITLE
Mint tokens for your community

### PR DESCRIPTION
**Pull request by jorge-campo**
Hey @shamisem - Could you please review this article and check if something is missing or hard to follow?

Thanks!

### CODE REVIEW

File: docs/en/status-communities/mint-tokens-for-your-community.md [hunk 1]
```diff
@@ -8,9 +8,51 @@ hide:
...
+The token purpose and use are specific to your community. For example, you can reward your entire community or individual members with custom tokens, or offer [exclusive membership to token holders][grant-exclusive-access-with-custom-tokens]. After minting your tokens, you can distribute them to community and non-community members with [airdrops][how-to-airdrop-tokens-in-Status].
+
+!!! note
+    Currently, you can only mint collectibles. Minting assets will be available in a future Status release.
+
+When you mint a token, you create a new and unique digital asset in the blockchain network. The blockchain charges you a transaction fee for this process. To know more about blockchain fees, check out [Understand network fees][understand-network-fees].
+
+## What to expect
+
+- Community tokens you mint exist in the context of your community and don't appear in your [Status Wallet][status-wallet-your-quick-start-guide].
```
Comment (shamisem) [inline docs/en/status-communities/mint-tokens-for-your-community.md:25]: Two suggestions:
- 'Community tokens you mint exist in the context of your community': this sort of sounds as if we're talking about theme park tokens, i.e. tokens that have no value once you leave the theme park that issued them. Community tokens are still blockchain tokens like any other, though, and perhaps we'd want to emphasise this. You're much more knowledgeable on this topic, so I'm just thinking out loud - e.g. if there are ways to use those tokens outside of Status, it may make sense to rephrase this.
- I feel like a clarification is missing on as to why they won't show up in the Wallet and if there's a way to change that. 

As a possible wording: 'Community tokens you mint belong to your community and depend on it. This is why they don't appear in your Wallet automatically once you mint them on your community's behalf, but you can [airdrop] them to your or any other address.'

File: docs/en/status-communities/mint-tokens-for-your-community.md [hunk 2]
```diff
@@ -8,9 +8,51 @@ hide:
 
 # Mint tokens for your community
 
- :octicons-tools-24: In progress
-
 !!! note ""
-     We're working on this content.
+    Currently, you can only mint tokens using the Status desktop app. [:octicons-desktop-download-16: Download Status desktop][status-web-download].
+
+With Status, you can mint custom tokens as a community owner. Owners have complete control over token creation and distribution.
```
[the remark this answers is not in the document]
Comment (jorge-campo) [reply docs/en/status-communities/mint-tokens-for-your-community.md:14]: As we have many articles starting in the same way, I was changing a bit the way to address the user in this case. I'm applying your second suggestion. Thanks!

File: docs/en/status-communities/mint-tokens-for-your-community.md [hunk 3]
```diff
@@ -8,9 +8,51 @@ hide:
...
+
+The token purpose and use are specific to your community. For example, you can reward your entire community or individual members with custom tokens, or offer [exclusive membership to token holders][grant-exclusive-access-with-custom-tokens]. After minting your tokens, you can distribute them to community and non-community members with [airdrops][how-to-airdrop-tokens-in-Status].
+
+!!! note
+    Currently, you can only mint collectibles. Minting assets will be available in a future Status release.
+
+When you mint a token, you create a new and unique digital asset in the blockchain network. The blockchain charges you a transaction fee for this process. To know more about blockchain fees, check out [Understand network fees][understand-network-fees].
```
[the remark this answers is not in the document]
Comment (jorge-campo) [reply docs/en/status-communities/mint-tokens-for-your-community.md:21]: Maybe something to add to our style guide, if we want to make it consistent? I'm adding this to my list of updates.

File: docs/en/status-communities/mint-tokens-for-your-community.md [hunk 4]
```diff
@@ -8,9 +8,51 @@ hide:
...
+
+!!! note
+    You can't change the collectible description or options after minting.
+
+## Set up collectible options
+
+| Setting | Description |
+|:---|:---|
+| Token symbol | Choose up to four words to identify your token. Tokens within the same community can't have the same symbol. |
+| Account | Select the wallet account you want to use to mint the token and pay the transaction fees. |
```
[the remark this answers is not in the document]
Comment (jorge-campo) [reply docs/en/status-communities/mint-tokens-for-your-community.md:51]: Thanks, Emma. I won't apply this suggestion as I believe the original sentence make that point clear.

**Comment by jorge-campo** [comment]
Thanks, @shamisem !
